test/tstDetailsUtils.cpp

```diff
@@ -25,80 +24,10 @@
 #include <numeric>
 #include <vector>
 
-#define BOOST_TEST_MODULE StandardAlgorithms
+#define BOOST_TEST_MODULE Utils
```

Comment:
```suggestion
#define BOOST_TEST_MODULE SortingUtilities
```

Reply:
No, they are not just sorting utilities.

Reply:
`compute_offsets_in_sorted_view` ok